### PR TITLE
docs: Specify what configuration files need to be changed

### PR DESCRIPTION
**Pull request by tecfu**
Documentation is unclear about where these configuration values need to be changed. This PR fixes.

### CODE REVIEW

**Review by tinmarino — APPROVED** [review]

**Comment by tinmarino** [comment]
Seems OK to merge.

**Comment by tecfu** [comment]
> Seems OK to merge.

Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.

File: README.md [hunk 1]
```diff
@@ -149,12 +149,16 @@ $ jupyter qtconsole --generate-config
 
 Now you need to uncomment and change the following config options to `True`.
 
-For qtconsole:
+- qtconsole 
+
+Edit this following file: ~/.jupyter/jupyter_qtconsole_config.py
```
Comment (wmvanvliet) [inline README.md:154]:
```suggestion
Edit this following file: `~/.jupyter/jupyter_qtconsole_config.py`
```

File: README.md [hunk 2]
```diff
@@ -149,12 +149,16 @@ $ jupyter qtconsole --generate-config
...
 ```python
 c.ConsoleWidget.include_other_output = True
 ```
 
-For console:
+- console
+
+Edit the following file: ~/.jupyter/jupyter_console_config.py
```
Comment (wmvanvliet) [inline README.md:161]:
```suggestion
Edit the following file: `~/.jupyter/jupyter_console_config.py`
```

**Comment by wmvanvliet** [comment]
Thanks @tecfu!

**Pull request merged**